tdms/matlabio/doc/matlab_array.tex

```diff
@@ -1,18 +1,18 @@
-\documentclass[12pt,a4paper]{article}
+\documentclass[10pt,a4paper]{article}
 
 \usepackage[dvipdfm,final]{graphics,graphicx} % dvips graphics driver
 \author{Peter R.T. Munro}
 \title{Matlab and C array conversion}
 \begin{document}
 \maketitle
 This theory could be developed for a multidimensional array however I
-will restrict it to arrays of tow and threee dimensions. Matlab
+will restrict it to arrays of tow and three dimensions. Matlab
 constructs its arrays by laying them out row-wise in memory. This is
-depicted below in Figure 
-\begin{figure}[h]\includegraphics[width=1.\textwidth]{array.eps}
-%\begin{figure}[h]\includegraphics{array.eps}
-\caption{Layout of a matlab array in memory}
-\label{fig:matlab:array}
+depicted below in Figure
+\begin{figure}[h]
+  \includegraphics[width=1.\textwidth]{array.eps}
+  \caption{Layout of a matlab array in memory}
+  \label{fig:matlab:array}
```

Comment:
```suggestion
  \includegraphics[width=1.\textwidth]{array.eps}
  \caption{\label{fig:matlab:array} Layout of a matlab array in memory}
```

Figure labels can get lost if they're not put in the caption.

Reply:
ah don't you just love latex. (have incorporated this into [70e90e1](https://github.com/UCL/TDMS/pull/66/commits/70e90e1cb452f0770c92214f10d898828ac78311) because I noticed the figure wasn't actually inserted 🤦🏼 )